src/hooks/use-play-pause-replay.ts

```diff
@@ -17,7 +17,7 @@ interface UsePlayPauseReplayHook {
 
 export const usePlayPauseReplayHook = (): UsePlayPauseReplayHook => {
 	const [isFinished, setIsFinished] = useOneMSDelayedState(false);
-	const listener = useMediaStore(state => state.getListener());
+	const listener = useMediaStore(state => state.getListener)();
```

Comment:
Is this change on purpose?

Reply:
yes. calling `getListener` inside of `useMediaStore` will always cause different objects(and rerenders).
so we should get the function's ref from store (`getListener`)